src/templates/homepage.html

```diff
@@ -29,8 +30,31 @@ <h2 class="hero-subtitle">Simple yet flexible JavaScript charting for designers
 					<a class="button button__black" href="https://github.com/chartjs/Chart.js" target="_blank">GitHub</a>
 				</div>
 			</div>
-			<hr/>
+      <div class="feature">
```

Comment:
Shouldn't this be inside feature-container?

Reply:
Makes sense, fixed!